test/functional/tools/parameters/gx_select.xml

```diff
@@ -23,5 +23,15 @@ echo '$parameter' >> '$output'
                 </assert_contents>
             </output>
         </test>
+        <test>
+            <!-- Selects have an implicit first option default in tool tests.
+                 Do they in the actual API also?
+            -->
```

Comment:
```suggestion
            <!-- Selects implicitly default to the first option as the default option.
                    Additional verification in test_select_first_by_default API test.
            -->
```
I wonder if this is good behavior overall ? Say we're missing any valid options, when would we catch this during validation ?

Reply:
It is all green enough - I'll fix this comment as the first thing I do in my next model rev - I am sure a new PR will be open this week. Thanks